pkg/virt-launcher/virtwrap/agent_poller.go

```diff
@@ -0,0 +1,188 @@
+package virtwrap
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	"kubevirt.io/kubevirt/pkg/log"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
+)
+
+// TODO: make this configurable?
+const startDelay = 120
```

Comment:
not that good idea. you cannot presume how long the boot is going to take. also on resumes(once supported) it would be immediate

Reply:
removing
Done